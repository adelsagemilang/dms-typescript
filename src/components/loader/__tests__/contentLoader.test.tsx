import React from 'react';
import { shallow } from 'enzyme';
import ContentLoader from '../content';

test('ContentLoader renders correctly while props changed', () => {
  const wrapper = shallow(<ContentLoader height={400} width={400} />);
  expect(wrapper).toMatchSnapshot();

  wrapper.setProps({
    height: 300,
    width: 300,
  });
  expect(wrapper).toMatchSnapshot();
});
