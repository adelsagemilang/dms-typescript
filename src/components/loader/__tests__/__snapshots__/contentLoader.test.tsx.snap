// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentLoader renders correctly while props changed 1`] = `
<ContentLoader
  height={400}
  speed={2}
  width={400}
>
  <rect
    height={40}
    rx="3"
    ry="3"
    width={133.33333333333334}
    x="0"
    y="0"
  />
  <rect
    height={400}
    rx="3"
    ry="3"
    width={400}
    x="0"
    y="60"
  />
</ContentLoader>
`;

exports[`ContentLoader renders correctly while props changed 2`] = `
<ContentLoader
  height={300}
  speed={2}
  width={300}
>
  <rect
    height={30}
    rx="3"
    ry="3"
    width={100}
    x="0"
    y="0"
  />
  <rect
    height={300}
    rx="3"
    ry="3"
    width={300}
    x="0"
    y="60"
  />
</ContentLoader>
`;
